core/types/transaction_signing.go

```diff
@@ -69,12 +69,12 @@ func MakeSigner(config *params.ChainConfig, blockNumber *big.Int, blockTime uint
 // have the current block number available, use MakeSigner instead.
 func LatestSigner(config *params.ChainConfig) Signer {
 	if config.ChainID != nil {
+		if config.Cel2Time != nil {
+			return latestCeloSigner(config.ChainID, NewCancunSigner(config.ChainID))
+		}
 		if config.CancunTime != nil && !config.IsOptimism() {
 			return NewCancunSigner(config.ChainID)
 		}
-		if config.Cel2Time != nil {
-			return latestCeloSigner(config.ChainID, NewLondonSigner(config.ChainID))
-		}
 		if config.LondonBlock != nil {
```

Comment:
The reason was that we didn't want to allow signing 4844 transactions, which is done in the cancun signer.

Reply:
I wen back to the LondonSigner, but still moved the Cel2Time check to the top, so that we'll allow the Celo tx types (e.g. in the devnet where Cancun is now enabled).

Reply:
Sounds good, seems like this is a part that we need to adapt as the underlying geth code changes.